sh_script/fuzzing.sh

```diff
@@ -13,20 +13,22 @@ fuzz_folder=(
 
 show_prompt_info() {
     echo "Usage:"
-    echo "  -c Test case name, e.g. afl_pe, afl_all, libfuzzer_all"
+    echo "  -n Test case name, e.g. afl_pe, afl_all, libfuzzer_all"
     echo "  -t Excution time, default is 3600"
 	echo "  -b Build all test case for check"
+    echo "  -c Enable code coverage"
 	echo ""
 	echo "Example:"
-    echo "Run single test case:    bash fuzzing.sh -c afl_pe -t 3600"
-	echo "Run all test case:       bash fuzzing.sh -c afl_all -t 3600"
+    echo "Run single test case:    bash fuzzing.sh -n afl_pe -t 3600"
+	echo "Run all test case:       bash fuzzing.sh -n afl_all -t 3600"
```

Comment:
Nits: I know this is the minimal change, but it might be worth simplifying this in future to something like:

```bash
readonly script_name=${0##*/}

show_prompt_info() {
    cat <<-EOT
        -n Test case name, e.g. afl_pe, afl_all, libfuzzer_all
        blah, blah,
        blah
    EOT
}
```